Tests/ApolloPaginationTests/ConcurrencyTest.swift

```diff
@@ -75,11 +75,11 @@ final class ConcurrencyTests: XCTestCase {
   ) async {
     await withTaskGroup(of: Void.self) { group in
       let serverExpectation = Mocks.Hero.FriendsQuery.expectationForSecondPage(server: self.server)
-      group.addTask { try? await pager.loadMore() }
-      group.addTask { try? await pager.loadMore() }
-      group.addTask { try? await pager.loadMore() }
-      group.addTask { try? await pager.loadMore() }
-      group.addTask { try? await pager.loadMore() }
+      group.addTask { try? await pager.loadNext() }
+      group.addTask { try? await pager.loadNext() }
+      group.addTask { try? await pager.loadNext() }
+      group.addTask { try? await pager.loadNext() }
+      group.addTask { try? await pager.loadNext() }
 
       group.addTask { await self.fulfillment(of: [serverExpectation, expectation], timeout: 100) }
       await group.waitForAll()
```

Comment:
Nit: We usually put test helpers above the tests. And I think these three load data functions should be underneath the `// MARK: - Test Helpers` header as well.